### ISSUE TITLE
should not depends on jmh

### ISSUE THREAD

**Issue by xuwei-k** [non/jawn#33]
http://repo1.maven.org/maven2/org/spire-math/jawn-ast_2.11/0.8.0/jawn-ast_2.11-0.8.0.pom

```xml
<?xml version='1.0' encoding='UTF-8'?>
<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0">
    <modelVersion>4.0.0</modelVersion>
    <groupId>org.spire-math</groupId>
    <artifactId>jawn-ast_2.11</artifactId>
    <packaging>jar</packaging>
    <description>jawn-ast</description>
    <url>http://github.com/non/jawn/</url>
    <version>0.8.0</version>
    <licenses>
        <license>
            <name>MIT</name>
            <url>http://opensource.org/licenses/MIT</url>
            <distribution>repo</distribution>
        </license>
    </licenses>
    <name>jawn-ast</name>
    <organization>
        <name>org.spire-math</name>
        <url>http://github.com/non/jawn/</url>
    </organization>
    <developers>
        <developer>
            <id>non</id>
            <name>Erik Osheim</name>
            <url>http://github.com/non</url>
        </developer>
    </developers>
    <scm>
        <connection>scm:git:github.com/non/jawn.git</connection>
        <developerConnection>scm:git:git@github.com:non/jawn.git</developerConnection>
        <url>https://github.com/non/jawn</url>
    </scm>
    <dependencies>
        <dependency>
            <groupId>org.scala-lang</groupId>
            <artifactId>scala-library</artifactId>
            <version>2.11.6</version>
        </dependency>
        <dependency>
            <groupId>org.spire-math</groupId>
            <artifactId>jawn-parser_2.11</artifactId>
            <version>0.8.0</version>
        </dependency>
        <dependency>
            <groupId>org.openjdk.jmh</groupId>
            <artifactId>jmh-core</artifactId>
            <version>1.9.1</version>
        </dependency>
        <dependency>
            <groupId>org.openjdk.jmh</groupId>
            <artifactId>jmh-generator-bytecode</artifactId>
            <version>1.9.1</version>
        </dependency>
        <dependency>
            <groupId>org.openjdk.jmh</groupId>
            <artifactId>jmh-generator-reflection</artifactId>
            <version>1.9.1</version>
        </dependency>
        <dependency>
            <groupId>org.scalatest</groupId>
            <artifactId>scalatest_2.11</artifactId>
            <version>2.2.4</version>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>org.scalacheck</groupId>
            <artifactId>scalacheck_2.11</artifactId>
            <version>1.12.2</version>
            <scope>test</scope>
        </dependency>
    </dependencies>
</project>
```

**Comment by non** [comment]
Argh! Good catch, sorry!

I think there will be a quick 0.8.1 release where I can fix this.

**Comment by jdegoes** [comment]
@non This is quite important as JMH is GPL 2 licensed and is only used for the benchmark. Any word on when a fix might be available?

**Comment by non** [comment]
Sorry! This should be fixed in 0.8.3. It took me awhile to figure out how to prevent [sbt-jmh](https://github.com/ktoso/sbt-jmh) from adding this dependency everywhere.
